src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -109,7 +110,10 @@ export default {
 			return client.query({
 				query: userInfoQuery,
 			}).then(() => {
-				return client.query({ query: experimentQuery, variables: { id: EXP_KEY } });
+				return client.query(
+					{ query: experimentQuery, variables: { id: EXP_KEY } },
+					{ query: experimentQuery, variables: { id: LOAN_CARD_EXP_KEY } }
```

Comment:
I don't think that this is needed with the new `active_experiments` process.

Reply:
Both of them?

Reply:
I also wondered if it was needed, it isn't for assignment, but may be to ensure the readQuery can get the setting...

Reply:
Actually I just did this in my locally header-exp code, this assignment is ran in the ancestor page component, and I didn't need to run it in the header component.

Reply:
Yeah, Im getting the experiment data and cookie without this, agree removing both?

Reply:
Yup, just verify in dev (removing cookie + getting assignment) once it gets merged, but the behavior should work the same locally.

Reply:
We currently assign all active experiments in `experimentPreFetch.js`.

Reply:
ok, removed